dcos/config.py

```diff
@@ -170,12 +170,16 @@ def get_config_val_envvar(name, config=None):
 
     section, subkey = split_key(name.upper())
 
-    env_var = None
+    env_var = ''
     if section == "CORE":
         if subkey.startswith("DCOS") and os.environ.get(subkey):
             env_var = subkey
         else:
             env_var = "DCOS_{}".format(subkey)
+    elif section == "CLUSTER" and subkey == "NAME":
```

Comment:
I'm not a huge fan of special casing just this one ENV variable and ignoring it. Is there some more general solution to this problem?

Reply:
Thinking about all the env variables we want to support and explicitly whitelisting them would sound cleaner to me. I went for the quickfix here as this will get replaced soon during the 1.12 refactoring.